etna/records/models.py

```diff
@@ -174,6 +174,13 @@ def title(self) -> str:
             pass
         return self.get("summary.title", default="")
 
+    @cached_property
+    def is_tna(self):
```

Comment:
Could we have a test case for this new property?

Reply:
Unit tests added in b57a855 :)